BlazarUI/app/scripts/components/sidebar/BuildsSidebar.jsx

```diff
@@ -1,43 +1,49 @@
 import React, {Component, PropTypes} from 'react';
-import BuildsSidebarListItem from './BuildsSidebarListItem.jsx';
+import $ from 'jquery';
+import LazyRender from '../shared/LazyRender.jsx';
 import SidebarFilter from './SidebarFilter.jsx';
 import fuzzy from 'fuzzy';
-import {bindAll, filter, contains} from 'underscore';
+import {bindAll, filter, contains, has} from 'underscore';
 import SectionLoader from '../shared/SectionLoader.jsx';
-import LazyRender from '../shared/LazyRender.jsx';
 import Helpers from '../ComponentHelpers';
 import MutedMessage from '../shared/MutedMessage.jsx';
+import SidebarItem from '../sidebar/SidebarItem.jsx';
+import Immutable from 'immutable';
```

Comment:
There's no `package.json` dependency on `immutable`, but it looks like it isn't actually used here.